kmr_mg-ud-test.conllu

```diff
@@ -9442,8 +9442,8 @@
 6	gelek	gelek	DET	det	Number=Plur|PronType=Ind	7	det	_	_
 7	Spor	spor	NOUN	n	Case=Nom|Definite=Def|Gender=Fem|Number=Plur	10	nsubj	_	_
 8	hatin	hatin	AUX	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	10	aux	_	_
-9	pêk	pêk	ADV	adv	_	10	compound:lvc	_	_
-10	anîn	anîn	VERB	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No
+9	pêk  	pêk anîn 	VERB	vblex	Typo=Yes|Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	        root	    _	        SpaceAfter=No|CorrectForm=pêkanîn
+10	anîn	_           X       VB	    _                                                               9           goeswith    9:goeswith  CorrectForm=_|XML=</sic> 	                                                             _	_	    _	_
```

Comment:
@dan-zeman Could you check and specify if there is anything wrong. The goeswith doc does not cover every column.

Reply:
I am not sure if "VB" is correct and other columns

Reply:
> I am not sure if "VB" is correct and other columns

Hard to say. It is the XPOS column, the tagset is not regulated by UD guidelines. But it does look different than the other XPOS tags.

Reply:
> @dan-zeman Could you check and specify if there is anything wrong. The goeswith doc does not cover every column.

I am at an airport where the wifi won't let me git pull your branch. But as I wrote in another comment, it should be sorted out whether the spelling with the space can be considered correct under some conditions because otherwise there should be no exception for words with spaces, and the solution of the typos will be different.

Reply:
https://github.com/UniversalDependencies/UD_Kurmanji-MG/pull/4#discussion_r1667624375 

In here I stated which cases it is written together and which cases it is written seperatly